packages/core/src/js/feedback/FeedbackForm.tsx

```diff
@@ -44,6 +47,31 @@ export const showFeedbackForm = (navigation: Navigation): void => {
   }
 };
 
+/**
+ * @beta
+ * Implements a feedback button that opens the feedback form screen to Sentry using Sentry.captureFeedback.
+ */
+export class FeedbackButton extends React.Component<FeedbackFormProps> {
+  /**
+   *
+   */
+  public render(): React.ReactNode {
+    const text: FeedbackTextConfiguration = { ...defaultConfiguration, ...this.props };
+    const styles: FeedbackFormStyles = { ...defaultStyles, ...this.props.styles };
+
+    return (
+      <TouchableOpacity
+        style={styles.triggerButton}
+        onPress={() => showFeedbackForm(this.props?.navigation)}
+        accessibilityLabel={text.triggerAriaLabel}
+      >
+        <Image source={{ uri: feedbackIcon }} style={styles.triggerIcon}/>
+        <Text style={styles.triggerText}>{text.triggerLabel}</Text>
```

Comment:
i: it would be nice to have an extra space between the text and image
```suggestion
        <Text style={styles.triggerText}> {text.triggerLabel}</Text>
```
![image](https://github.com/user-attachments/assets/9d89a710-d496-48b0-b570-2e5c398b9d23)

Reply:
vs before 
![image](https://github.com/user-attachments/assets/aa1921e4-bb68-422b-85b8-0c18dd633508)

Reply:
Good catch @lucas-zimerman 👍 
I missed that on Android 😓 
I updated the code with 0eb4f68a6093e8a5c96aa73595885479362f1713 to add a margin instead of a space. Wdyt? 

|before|After|
|---|---|
|<img width="729" alt="Screenshot 2024-12-18 at 4 59 32 PM" src="https://github.com/user-attachments/assets/eb3a3262-3fc2-4b39-bd97-e23aae9a87c1" />|<img width="753" alt="Screenshot 2024-12-18 at 5 02 42 PM" src="https://github.com/user-attachments/assets/557d4cd1-301a-4397-9d3d-1749987e691c" />|